board/batocera/amlogic/vim3/create-boot-script.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# HOST_DIR = host dir
+# BOARD_DIR = board specific dir
+# BUILD_DIR = base dir/build
+# BINARIES_DIR = images dir
+# TARGET_DIR = target dir
+# BATOCERA_BINARIES_DIR = batocera binaries sub directory
+
+HOST_DIR=$1
+BOARD_DIR=$2
+BUILD_DIR=$3
+BINARIES_DIR=$4
+TARGET_DIR=$5
+BATOCERA_BINARIES_DIR=$6
+
+mkdir -p "${BATOCERA_BINARIES_DIR}/boot/boot/extlinux" || exit 1
+
+cp "${BINARIES_DIR}/Image"                             "${BATOCERA_BINARIES_DIR}/boot/boot/linux"           || exit 1
+cp "${BINARIES_DIR}/initrd.gz"                         "${BATOCERA_BINARIES_DIR}/boot/boot/initrd.gz"       || exit 1
+cp "${BINARIES_DIR}/rootfs.squashfs"                   "${BATOCERA_BINARIES_DIR}/boot/boot/batocera.update" || exit 1
+
+cp "${BINARIES_DIR}/meson-g12b-a311d-khadas-vim3.dtb"  "${BATOCERA_BINARIES_DIR}/boot/boot/"          || exit 1
+cp "${BINARIES_DIR}/boot.scr"                          "${BATOCERA_BINARIES_DIR}/boot/"               || exit 1
+cp "${BOARD_DIR}/boot/logo.bmp"                        "${BATOCERA_BINARIES_DIR}/boot/boot/"          || exit 1
+cp "${BOARD_DIR}/boot/extlinux.conf"                   "${BATOCERA_BINARIES_DIR}/boot/boot/extlinux/" || exit 1
+
+#dd if=${BINARIES_DIR}/u-boot.bin.sd.bin of=${BATOCERAIMG} conv=fsync,notrunc bs=512 skip=1 seek=1 || exit 1
```

Comment:
I couldn't find a way to put this in genimage. I can create two file fragments here and use them in genimage. Nevertheless, two dd commands will remain here..